### PR TITLE
Feature/add eh refresh colours

### PR DESCRIPTION
**Pull request by marymooney**
Ticket: https://webjobs.agepartnership.co.uk/desk/tickets/5497817/messages

Changes: Adds colours & typography required for the Eamonn Holmes email refresh. I wasn't 100% sure about the approach to colours. Didn't seem to make sense having grey-1 and grey-2 so I have removed but I imagine when we start on the site rebuild these will be re-jigged / added. 

It's fairly rough but you can test on this branch - https://github.com/AgePartnership/mjolnir-templates/compare/feature/RefreshDay0 , checkout then run npm install git+ssh://github.com/AgePartnership/oleg-brand-kit.git#feature/AddEHRefreshColours and then run gulp  and gulp watch to preview the day0 template http://localhost:8080/templates/emails/age-partnership/eqr/welcome/day-0.html.twig

Checks: 
- [ ] Meets code standards / guidelines
- [ ] New styles compile correctly
- [ ] Malicious code

### CODE REVIEW

**Review by beccadawson — APPROVED** [review]
Approving as sensible and non-malicious, I'm just not sure how we know the font sizes / line heights as they've not specified afaict?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "oleg-brand-kit",
-  "version": "2.0.8",
+  "version": "2.0.9",
```
Comment (beccadawson) [inline package.json:3]: Minor version is the middle number! `Major.Minor.Patch`

**Review by beccadawson — CHANGES_REQUESTED** [review]

**Review by beccadawson — APPROVED** [review]

**Comment by marymooney** [comment]
@beccadawson 
> Approving as sensible and non-malicious, I'm just not sure how we know the font sizes / line heights as they've not specified afaict?

Do you want me to ask ATP? As they haven't specified I'd gone with basing it off our existing font sizes/line heights. I thought font sizes was one of the things we weren't pushing them too hard on but maybe I misunderstood!

**Comment by beccadawson** [comment]
> @beccadawson
> 
> > Approving as sensible and non-malicious, I'm just not sure how we know the font sizes / line heights as they've not specified afaict?
> 
> Do you want me to ask ATP? As they haven't specified I'd gone with basing it off our existing font sizes/line heights. I thought font sizes was one of the things we weren't pushing them too hard on but maybe I misunderstood!

No its fine, I just wasn't sure!

**Pull request merged**